.github/workflows/pull-request-commentor.yaml

```diff
@@ -27,10 +27,6 @@ jobs:
           - k8s: "1.26"
             branch: "devel"
 
-          # Ceph-CSI <= 3.11 was released before Kubernetes 1.30
-          - k8s: "1.30"
-            branch: "release-v3.11"
-
```

Comment:
Sorry for a mistake from my side, now i see that we don't support 1.30 with even patch releases. can you please remove 1.30 check from mergify.yaml where we have `base~=^(release-.+)$` check and nothing else.